portfolio/src/main/webapp/script.js

```diff
@@ -12,17 +12,45 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-/**
- * Adds a random greeting to the page.
+/*
+ * Adds a random activity car to the page.
```

Comment:
Remove the word "car"?
Also I would make the comment a little more descriptive to say what exactly these activities will be used for and such.

Reply:
Was meant to say "card" which is the bootstrap term for the popup these come in. Will add more to the comment to describe.